plugins/module_utils/network/sonic/config/vrfs/vrfs.py

```diff
@@ -418,3 +425,19 @@ def get_replaced_config(self, have, want):
                 replaced_vrfs.append(have_vrf)
 
         return replaced_vrfs
+
+    def validate_mgmt_vrf_in_want(self, want):
```

Comment:
Interfaces (Ethernet or port channel) can be added or deleted from a management VRF.

The management VRF can not, however, be deleted if it contains any non-management interfaces. (These interfaces must be removed before deleting the management VRF.)

Reply:
Thanks for correction. The change handled the case.